pkg/eks/nodegroup_service.go

```diff
@@ -273,3 +274,15 @@ func ValidateExistingNodeGroupsForCompatibility(ctx context.Context, cfg *api.Cl
 
 	return nil
 }
+func isWindowsAMIFamily(amiFamilyType string) bool {
+	switch amiFamilyType {
+	case api.NodeImageFamilyWindowsServer2019CoreContainer,
+		api.NodeImageFamilyWindowsServer2019FullContainer,
+		api.NodeImageFamilyWindowsServer2022CoreContainer,
+		api.NodeImageFamilyWindowsServer2022FullContainer:
+		return true
+
+	default:
+		return false
+	}
+}
```

Comment:
This function already exists in `validation.go` and can be imported here without introducing circular imports. 
```suggestion
```